### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Kanane9**

### CODE REVIEW

File: backstopConfig.js [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
-const backstop = require('@mate-academy/backstop-config');
+const backstop = require("@mate-academy/backstop-config");
 const { basicScenario } = backstop;
 
 const basic = {
   ...basicScenario,
-  label: 'Elementary test',
-  referenceUrl: basicScenario.referenceUrl + '/html-form/',
+  div: "Elementary test",
+  referenceUrl: basicScenario.referenceUrl + "/html-form/",
```
Comment (illiarerun) [inline backstopConfig.js:10]: Do not change the test files

File: checklist.md [hunk 2]
```diff
@@ -1,66 +1,59 @@
 1. [CODE STYLE] - Keep your code line length below 80. It’s not only historical
-tradition, but also allows your code to fit into one standard screen without
-horizontal scroll. 
-But do not break the line if it cannot be broken (ex., long links).
-
+   tradition, but also allows your code to fit into one standard screen without
+   horizontal scroll.
+   But do not break the line if it cannot be broken (ex., long links).
```
Comment (illiarerun) [inline checklist.md:5]: do not change checklists too

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,169 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
```
Comment (illiarerun) [inline src/index.html:4]: There should be no blank lines between parent and child blocks
```suggestion
<html lang="en">
  <head>
```

File: src/index.html [hunk 4]
```diff
@@ -1,17 +1,169 @@
...
   </head>
+
   <body>
     <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript"
+      src="./main.js"></script>
+    <form action="http://localhost:8080/api"
+      method="post">
+      <form>
```
Comment (illiarerun) [inline src/index.html:21]: Why do you need 2 forms?

Comment (illiarerun) [reply src/index.html:21]: Also the action `value` is wrong, you can get it from README

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,169 @@
...
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="form-field">
+            <label>Surname:
+              <input type="text"
+                name="userSurname"
+                required
+                autocomplete="off">
```
Comment (illiarerun) [inline src/index.html:29]: Check your code according to [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and fix all of your tags

File: src/style.css [hunk 6]
```diff
@@ -1 +1,12 @@
-/* styles go here */
+/* stylelint-disable CssSyntaxError */
+.form-field {
+  margin-bottom: 10px;
+}
+
+.bottom {
+  margin-bottom: 0;
+}
```
Comment (illiarerun) [inline src/style.css:8]: You can use `:last-child` pseudoclass to set styles for the last element

File: src/style.css [hunk 7]
```diff
@@ -1 +1,12 @@
...
+
+fieldset {
```
Comment (illiarerun) [inline src/style.css:10]: Use classes to set your styles, not the tag selectors

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start 👍 But let's work on fixes now:

File: checklist.md (hunk 2 above)
Comment (Kanane9) [reply checklist.md:5]: oh my, ctrl + f is realy powerfull...

**Pull request closed without merging**